core/src/main/java/ai/timefold/solver/core/api/domain/entity/PlanningEntity.java

```diff
@@ -37,18 +37,23 @@
     /**
      * A pinned planning entity is never changed during planning,
      * this is useful in repeated planning use cases (such as continuous planning and real-time planning).
-     * <p>
      * This applies to all the planning variables of this planning entity.
-     * To pin individual variables, see https://issues.redhat.com/browse/PLANNER-124
      * <p>
      * The method {@link PinningFilter#accept(Object, Object)} returns false if the selection entity is pinned
```

Comment:
Should it return true if the selection entity is pinned?

Reply:
This is an old class, which I'm now deprecating.
We can not change its behavior due to bwd compat, so even though we may have originally designed it better, there's no point discussing it now.